CHANGELOG.md

```diff
@@ -1,13 +1,244 @@
-## master / unreleased
+## 1.4.0 (2020-03-12)
 
-### Features
+Over 130 PR's merged for this release, from 40 different contributors!!  We continue to be humbled and thankful for the growing community of contributors and users of Loki.  Thank you all so much.
+
+### Important Notes
+
+**Really, this is important**
+
+Before we get into new features, version 1.4.0 brings with it the first (that we are aware of) upgrade dependency.
+
+We have created a dedicated page for upgrading Loki in the [operations section of the docs](https://github.com/grafana/loki/tree/v1.4.0/docs/operations/upgrade.md)
+
+Also you should be aware we are now pruning old `master-xxxxx` docker images from docker hub, currently anything older than 90 days is removed.  **We will never remove released versions of Loki**
```

Comment:
I dislike that we do this, but we've been having some trouble here so I understand. Hopefully our problem here disappears in the future.